todoman/cli.py

```diff
@@ -48,6 +48,24 @@ def _validate_date_param(ctx, param, val):
         raise click.BadParameter(e)
 
 
+def _validate_start_date_param(ctx, param, val):
+    if val is None:
+        return val
+    if val.startswith('before'):
```

Comment:
Two things:

* We need to check for the string `"before "`, not `"before"` (space at the end).
* `val` isn't modified, so further down you try to pass a string like `"before 2016-09-09"` to `parse_date`. Something like this is missing: `val = val[len("before "):]`

I suspect this is why the tests fail